.github/workflows/build-cache.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and cache site
+
+on:
+  push:
+    branches:
+      - develop
+
+env:
+  NODE_OPTIONS: '--max-old-space-size=4096'
+  CHOKIDAR_USEPOLLING: 1
```

Comment:
We can remove this for now if you'd like. We needed this on the docs site for the sheer amount of content there. It was running out of memory on builds. The dev site is much smaller and shouldn't suffer that problem for quite a while.

Reply:
I removed it in both files!